web/src/integration_url_modal.ts

```diff
@@ -174,6 +231,25 @@ export function show_generate_integration_url_modal(api_key: string): void {
             event.preventDefault();
             event.stopPropagation();
         }
+
+        function set_events_param(params: URLSearchParams): (string | number)[] {
```

Comment:
Looking at this return value and how we use it above, maybe this should be a boolean that's true if any events were set in the URL and false otherwise.